reference/docs-conceptual/overview.md

```diff
@@ -1,206 +1,64 @@
 ---
-ms.date: 05/22/2020
+ms.date: 03/22/2021
 keywords:  powershell,cmdlet
 title:  What is PowerShell?
 description: This article is an introduction to the PowerShell scripting environment and its features.
 ---
 
 # What is PowerShell?
 
-PowerShell is a cross-platform task automation and configuration management framework, consisting of
-a command-line shell and scripting language. Unlike most shells, which accept and return text,
-PowerShell is built on top of the .NET Common Language Runtime (CLR), and accepts and returns .NET
-objects. This fundamental change brings entirely new tools and methods for automation.
+PowerShell is a cross-platform task automation solution made up of a command-line shell, a scripting
+language, and a configuration management framework. PowerShell runs on Windows, Linux, and macOS.
 
-<!-- removing images until we can get replacements
-:::row:::
-   :::column span="":::
-      Windows
-      [![PowerShell on Windows](media/overview/windows-desktop-660.gif)](media/overview/windows-desktop.gif#lightbox)
-      [Install on Windows](install/installing-powershell-core-on-windows.md)
-   :::column-end:::
-   :::column span="":::
-      Linux
-      [![PowerShell on Linux](media/overview/linux-desktop-660.gif)](media/overview/linux-desktop.gif#lightbox)
-      [Install on Linux](install/installing-powershell-core-on-linux.md)
-   :::column-end:::
-   :::column span="":::
-      macOS
-      [![PowerShell on macOS](media/overview/macos-desktop-660.gif)](media/overview/macos-desktop.gif#lightbox)
-      [Install on macOS](install/installing-powershell-core-on-macos.md)
-   :::column-end:::
-:::row-end:::
--->
+## Shell
 
-## Output is object-based
+PowerShell is modern command shell that includes all of the best features of other popular shells.
```

Comment:
The only problem I have with this section is it's subjective and absolute. "...**_all_** of the **_best features_**...""

It might be better as "...includes many features of other poplular shells. "

Reply:
This was marked as resolved. just wanted to make sure I didn't miss anything. Do you think that this is the best verbiage we should use?

Reply:
I changed the wording.